recipes/trisbm/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+libXcomposite
```

Comment:
It's still here

Reply:
Ah maybe I wasn't clear, I mean the whole recipes/trisbm/yum_requirements.txt file isn't needed